Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -84,6 +86,11 @@ public class ComputeCredential : ServiceCredential, IOidcTokenProvider, IGoogleC
         /// </summary>
         public string OidcTokenUrl { get; }
 
+        /// <summary>
+        /// The explicitly set universe domain. May be null in which case the GCE universe domain will be used.
```

Comment:
GCE or GCP? (I didn't think GCE had its own universe domain.)

Reply:
Having read further comments, I think this really means "May be null, in which case the universe domain will be fetched from the metadata server" - is that correct?